buildSrc/src/main/kotlin/custom/Auth.kt

```diff
@@ -82,31 +82,29 @@ class Transaction(
     ).execute().prettyPrint()
 
   @Suppress("DEPRECATION")
-  fun attachExpansionTo(expansionCode: Int, apkVariantOutput: ApkVariantOutput): ExpansionFile =
+  fun attachExpansionTo(expansionCode: Int?, variantOutput: VariantOutput): ExpansionFile =
     publisher.edits().expansionfiles().update(
       packageName,
       editId,
-      apkVariantOutput.versionCodeOverride,
+      variantOutput.versionCode.get(),
       "main",
       ExpansionFile().apply { referencesVersion = expansionCode }
     ).execute().prettyPrint()
 
-  @Suppress("DEPRECATION")
-  fun uploadApk(apkVariantOutput: ApkVariantOutput) {
+  fun uploadApk() {
     publisher.edits().apks().upload(
       packageName,
       editId,
-      FileContent("application/octet-stream", apkVariantOutput.outputFile)
+      null
```

Comment:
Why we are not uploading the file here?

Reply:
I think it's redundant and the function uses `ApkVariantOutput` which is obsolete and the current API `variantOutput` lacks this functionality. Let me know what you think.

Reply:
@MohitMaliFtechiz I haven't got the chance to test it but I saw the documentation, I think it's redundant. Please Let me know if this is regression or bug?

Reply:
@gouri-panda, We are uploading apk variants for the different arch for custom apps with expansion files and this line uploads different arch apk to play store, but if we do not upload this then how will the apk upload to the Play Store? This is my concern here because apart from this method we are not uploading the apk anywhere.